app/templates/direct-award/save-search.html

```diff
@@ -33,36 +33,67 @@
           {% endwith %}
         </div>
-        <h2 class="heading-xmedium">Create a new saved search</h2>
-        <p>Name your search. A short description of what you want to buy makes a good name.</p>
-        {% if form.name.errors %}
-        <div class="validation-wrapper">
-            {% endif %}
-            <div class="question" id="{{ form.name.name }}">
-                {{ form.name.label(class="question-heading") }}
-                <p class="hint">
-                    100 characters maximum
-                </p>
-                {% if form.name.errors %}
-                <p class="validation-message" id="error-name-textbox">
-                    {% for error in form.name.errors %}{{ error }}{% endfor %}
-                </p>
-                {% endif %}
-                {{ form.name(class="text-box") }}
-            </div>
-            {% if form.name.errors %}
+
+        <div class="dmspeak">
+            <h2 class="heading-xmedium">Choose where to save your search</h2>
         </div>
-        {% endif %}
+        
+        <fieldset>
+            <div class="multiple-choice" data-target="new-search">
+                <input type="radio" name="save_search_selection" value="new_search" {% if request.form.save_search_selection == 'new_search' or not projects %}checked="checked"{% endif %}>
+                <label for="new-search">Create a new search</label>
+              </div>
+              <div class="panel panel-border-narrow js-hidden" id="new-search">
+                  {% if form.name.errors %}
+                  <div class="validation-wrapper">
+                      {% endif %}
+                      <div class="question" id="{{ form.name.name }}">
+                          {{ form.name.label(class="question-heading") }}
+                          <p>Name your search. A short description of what you want to buy makes a good name.</p>
+                          <p class="hint">
+                              100 characters maximum
+                          </p>
+                          {% if form.name.errors %}
+                          <p class="validation-message" id="error-name-textbox">
+                              {% for error in form.name.errors %}{{ error }}{% endfor %}
+                          </p>
+                          {% endif %}
+                          {{ form.name(class="text-box") }}
+                      </div>
+                      {% if form.name.errors %}
+                  </div>
+                  {% endif %}
+              </div>
+
+              {% if projects %}
+              
+              <p class="form-block">or</p>
+
+              {% for project in projects %}
+
+              <div class="multiple-choice">
+                <input type="radio" name="save_search_selection" value="{{ project.id }}" {% if request.form.save_search_selection == project.id %}checked="checked"{% endif %}>
+                <label for="exisitng-search">{{ project.name or "No Project Name" }}</label>
```

Comment:
There's a typo here, but in any event the label needs to be associated with the _specific_ input element, otherwise it won' work.

Potentially the easiest solution is to drop the `for` attribute altogether, and just wrap the `input` inside the `label`.